Client/ButtonAction.cpp

```diff
@@ -73,6 +74,16 @@ void selectAWritable(World &world, Entity &entityPtr)
         return;
     }
     resource.window.setKeyRepeatEnabled(true);
+    auto joined = world.joinEntities<Selected, WritableContent>();
+    if (!joined.empty()) {
+        auto guardOld = std::lock_guard(*(joined.at(0)));
+        joined.at(0)->removeComponent<Selected>();
+        if (joined.at(0)->contains<TextureName>()) {
+            joined.at(0)->removeComponent<TextureName>();
+            joined.at(0)->addComponent<TextureName>(GraphicsTextureResource::WRITABLE);
```

Comment:
Instead of removing and adding, just change the value

Reply:
:eyes: